molecule-runtime/src/commonMain/kotlin/app/cash/molecule/molecule.kt

```diff
@@ -95,6 +116,32 @@ private fun <T> immediateClockFlow(body: @Composable () -> T): Flow<T> = flow {
   }
 }
 
+/**
+ * Launch a coroutine into this [CoroutineScope] which will continually recompose `emittingBody`
+ * to produce a [StateFlow] stream of [T] values.
+ */
+public fun <T> CoroutineScope.launchMolecule(
+  mode: RecompositionMode,
+  emittingBody: @Composable (Emitter<T>) -> Unit,
+): StateFlow<T> {
+  var flow: MutableStateFlow<T>? = null
+
+  launchMolecule(
+    mode = mode,
+    emitter = { value ->
+      val outputFlow = flow
+      if (outputFlow != null) {
+        outputFlow.value = value
+      } else {
+        flow = MutableStateFlow(value)
+      }
+    },
+    body = emittingBody,
+  )
+
+  return flow!!
```

Comment:
There is no guarantee the composition invoked the emitter so a `StateFlow` cannot be exposed. It has to be a `Flow`.